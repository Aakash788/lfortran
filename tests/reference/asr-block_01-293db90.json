{
    "basename": "asr-block_01-293db90",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/block_01.f90",
    "infile_hash": "e1008df68ca08f26a1d9f91a68ee7ec4e7744837a022b483c129a5c1",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-block_01-293db90.stderr",
    "stderr_hash": "44cf2d3a65bdeb0cad89e7c98edcb4f88af5c55fbbc032bea829cdd2",
    "returncode": 1
}