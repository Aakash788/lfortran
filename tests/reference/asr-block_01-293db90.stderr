Traceback (most recent call last):
[2m  Binary file "[0m[1m[35m$DIR/src/bin/lfortran[39m[0m[2m", in _start()[0m
[2m  File "[0m[1m[35m./csu/../csu/libc-start.c[39m[0m[2m", line 381, in __libc_start_main_impl()[0m
[2m  File "[0m[1m[35m./csu/../sysdeps/nptl/libc_start_call_main.h[39m[0m[2m", line 58, in __libc_start_call_main()[0m
[2m  File "[0m[1m[35m$DIR/src/bin/lfortran.cpp[39m[0m[2m", line 1864, in main()[0m
    return emit_asr(arg_file, lfortran_pass_manager,
[2m  File "[0m[1m[35m$DIR/src/bin/lfortran.cpp[39m[0m[2m", line 638, in emit_asr()[0m
    std::cout << LCompilers::LFortran::pickle(*asr, compiler_options.use_colors, compiler_options.indent,
[2m  File "[0m[1m[35m$DIR/src/lfortran/pickle.cpp[39m[0m[2m", line 289, in LCompilers::LFortran::pickle[abi:cxx11](LCompilers::ASR::TranslationUnit_t&, bool, bool, bool)[0m
    return pickle((ASR::asr_t &)asr, colors, indent, show_intrinsic_modules);
[2m  File "[0m[1m[35m$DIR/src/lfortran/pickle.cpp[39m[0m[2m", line 284, in LCompilers::LFortran::pickle[abi:cxx11](LCompilers::ASR::asr_t&, bool, bool, bool)[0m
    v.visit_asr(asr);
[2m  File "[0m[1m[35m$DIR/src/libasr/asr.h[39m[0m[2m", line 4540, in LCompilers::ASR::BaseVisitor<LCompilers::LFortran::ASRPickleVisitor>::visit_asr(LCompilers::ASR::asr_t const&)[0m
    void visit_asr(const asr_t &b) { visit_asr_t(b, self()); }
[2m  File "[0m[1m[35m$DIR/src/libasr/asr.h[39m[0m[2m", line 4516, in visit_asr_t<LCompilers::LFortran::ASRPickleVisitor>()[0m
    case asrType::unit: { v.visit_unit((const unit_t &)x); return; }
[2m  File "[0m[1m[35m$DIR/src/libasr/asr.h[39m[0m[2m", line 4541, in LCompilers::ASR::BaseVisitor<LCompilers::LFortran::ASRPickleVisitor>::visit_unit(LCompilers::ASR::unit_t const&)[0m
    void visit_unit(const unit_t &b) { visit_unit_t(b, self()); }
[2m  File "[0m[1m[35m$DIR/src/libasr/asr.h[39m[0m[2m", line 4272, in visit_unit_t<LCompilers::LFortran::ASRPickleVisitor>()[0m
    case unitType::TranslationUnit: { v.visit_TranslationUnit((const TranslationUnit_t &)x); return; }
[2m  File "[0m[1m[35m$DIR/src/libasr/asr.h[39m[0m[2m", line 14158, in LCompilers::ASR::PickleBaseVisitor<LCompilers::LFortran::ASRPickleVisitor>::visit_TranslationUnit(LCompilers::ASR::TranslationUnit_t const&)[0m
    this->visit_symbol(*a.second);
[2m  File "[0m[1m[35m$DIR/src/libasr/asr.h[39m[0m[2m", line 4543, in LCompilers::ASR::BaseVisitor<LCompilers::LFortran::ASRPickleVisitor>::visit_symbol(LCompilers::ASR::symbol_t const&)[0m
    void visit_symbol(const symbol_t &b) { visit_symbol_t(b, self()); }
[2m  File "[0m[1m[35m$DIR/src/libasr/asr.h[39m[0m[2m", line 4280, in visit_symbol_t<LCompilers::LFortran::ASRPickleVisitor>()[0m
    case symbolType::Program: { v.visit_Program((const Program_t &)x); return; }
[2m  File "[0m[1m[35m$DIR/src/libasr/asr.h[39m[0m[2m", line 14241, in LCompilers::ASR::PickleBaseVisitor<LCompilers::LFortran::ASRPickleVisitor>::visit_Program(LCompilers::ASR::Program_t const&)[0m
    this->visit_symbol(*a.second);
[2m  File "[0m[1m[35m$DIR/src/libasr/asr.h[39m[0m[2m", line 4543, in LCompilers::ASR::BaseVisitor<LCompilers::LFortran::ASRPickleVisitor>::visit_symbol(LCompilers::ASR::symbol_t const&)[0m
    void visit_symbol(const symbol_t &b) { visit_symbol_t(b, self()); }
[2m  File "[0m[1m[35m$DIR/src/libasr/asr.h[39m[0m[2m", line 4292, in visit_symbol_t<LCompilers::LFortran::ASRPickleVisitor>()[0m
    case symbolType::AssociateBlock: { v.visit_AssociateBlock((const AssociateBlock_t &)x); return; }
[2m  File "[0m[1m[35m$DIR/src/libasr/asr.h[39m[0m[2m", line 15288, in LCompilers::ASR::PickleBaseVisitor<LCompilers::LFortran::ASRPickleVisitor>::visit_AssociateBlock(LCompilers::ASR::AssociateBlock_t const&)[0m
    self().visit_stmt(*x.m_body[i]);
[2m  File "[0m[1m[35m$DIR/src/libasr/asr.h[39m[0m[2m", line 4558, in LCompilers::ASR::BaseVisitor<LCompilers::LFortran::ASRPickleVisitor>::visit_stmt(LCompilers::ASR::stmt_t const&)[0m
    void visit_stmt(const stmt_t &b) { visit_stmt_t(b, self()); }
[2m  File "[0m[1m[35m$DIR/src/libasr/asr.h[39m[0m[2m", line 4338, in visit_stmt_t<LCompilers::LFortran::ASRPickleVisitor>()[0m
    case stmtType::BlockCall: { v.visit_BlockCall((const BlockCall_t &)x); return; }
[2m  File "[0m[1m[35m$DIR/src/libasr/asr.h[39m[0m[2m", line 17181, in LCompilers::ASR::PickleBaseVisitor<LCompilers::LFortran::ASRPickleVisitor>::visit_BlockCall(LCompilers::ASR::BlockCall_t const&)[0m
    self().visit_symbol(*x.m_m);
[2m  File "[0m[1m[35m$DIR/src/lfortran/pickle.cpp[39m[0m[2m", line 214, in LCompilers::LFortran::ASRPickleVisitor::visit_symbol(LCompilers::ASR::symbol_t const&)[0m
    s.append(ASRUtils::symbol_name(&x));
[2m  Binary file "[0m[1m[35m/lib/x86_64-linux-gnu/libstdc++.so.6[39m[0m[2m", in std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::append(char const*)[0m
[2m  File "[0m[1m[35m./string/../sysdeps/x86_64/multiarch/strlen-avx2.S[39m[0m[2m", line 73, in __strlen_avx2()[0m
[2m  Binary file "[0m[1m[35m/lib/x86_64-linux-gnu/libc.so.6[39m[0m[2m", in __GI___sigaction()[0m
Segfault: Signal SIGSEGV (segmentation fault) received
